docs/self-managed/concepts/authentication/authentication-to-management-components.md

```diff
@@ -0,0 +1,61 @@
+---
+id: authentication-to-management-components
+title: Authentication to Management Components on Self-Managed
+sidebar_label: "Authentication to Management Components"
+description: "Learn about authentication methods for the Management Components on Self-Managed and how to choose the right one for your environment."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Authentication to the Camunda 8 management components and their resources is managed by [Management Identity](/self-managed/components/management-identity/overview.md). This includes components like [Web Modeler](/self-managed/components/modeler/web-modeler/overview.md), [Console](/self-managed/components/console/overview.md), and [Optimize](/self-managed/components/optimize/overview.md).
```

Comment:
I think they are called Management and Modeling components according to the architecture diagram